### PR TITLE
Pr for machine learning task

### PR DESCRIPTION
**Pull request by kzcyanide**

### CODE REVIEW

File: Machine Learning/Task Submission/krishna.ipynb [hunk 1]
```diff
@@ -0,0 +1,728 @@
...
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "WCl1osIAMOaa"
+      },
+      "source": [
+        "#here we split our data into training and testing data\n",
+        "x=data.drop(columns=['class','sepal_width'])\n",
```
Comment (sambit221) [inline Machine Learning/Task Submission/krishna.ipynb:505]: Dropping some other feature along with target feature may can alter the predicted value. 
you don't have to drop the 'sepal_width' feature. Correct it up and commit again.

Comment (Dev-Neon221) [inline Machine Learning/Task Submission/krishna.ipynb]: Dropping some other feature along with target feature may can alter the predicted value.
you don't have to drop the 'sepal_width' feature. Correct it up and commit again.

**Review by Rockzzzz-221 — CHANGES_REQUESTED** [review]
Correct it up as mentioned and commit again.

File: Machine Learning/Task Submission/krishna.ipynb (hunk 1 above)
Comment (kzcyanide) [reply Machine Learning/Task Submission/krishna.ipynb]: but 'sepal_width ' feature has -ve correlation with other features and adding it doesn't increase the accuracy of our model
i should have mentioned it in a comment  my bad

**Review by Rockzzzz-221 — APPROVED** [review]

**Pull request merged**